torch/csrc/distributed/rpc/request_callback_impl.cpp

```diff
@@ -369,7 +369,6 @@ void RequestCallbackImpl::processPythonRemoteCall(
           IValue py_ivalue = jit::toIValue(result, PyObjectType::get());
 
           py::gil_scoped_release release;
-          ownerRRef->recordAllStreams(lsctx);
```

Comment:
`recordAllStreams` recorded a new event in each of `lsctx`'s streams. However, in the line just below, we call `setValue` which calls `markCompleted` on the future, which does the same thing already.

Reply:
If we remove this, I think we will need to pass context devices to `getOrCreateOwnerRRef` on line 338, so that the created RRef will use CUDA ivalue::Future.